signer/src/transaction_coordinator.rs

```diff
@@ -367,9 +370,11 @@ where
 
         self.construct_and_sign_stacks_sbtc_response_transactions(
             &bitcoin_chain_tip,
+            &wallet,
             &aggregate_key,
         )
         .await?;
+        tracing::debug!("processed new blocks successfully");
```

Comment:
Maybe something more like:

```suggestion
        tracing::info!("coordinator tenure completed successfully");
```

Reply:
Updated in https://github.com/stacks-network/sbtc/pull/1403/commits/a563ae01396f4c1e99c75d6ab6463dfb6ab12adb